src/Runner.Worker/Variables.cs

```diff
@@ -165,6 +165,21 @@ public DictionaryContextData ToSecretsContext()
             }
             return result;
         }
+
+        public DictionaryContextData ToVarsContext()
+        {
+            var result = new DictionaryContextData();
+            foreach (var variable in _variables.Values)
```

Comment:
I am wondering whether we should not build this new feature on top of the `Variables` object.
The `Variables` is a legacy thing only used for sending secrets and system variables.

Reply:
We used the `Variables` object as it used to handle both secrets and plain-text variables in case of AzureDevops. 

I am not sure what it would men to introduce a new field like `ConfigVariables` in runner and the Actions Service. I believe the variables are right now stored as part of the plan.

Reply:
I tried to pass `vars` as part of the context object. The context was passed and not explicit handling was required.

Reply:
@TingluoHuang Both Action's service and runner PRs are updated to use the context to pass `vars`.
Could you please review again.